index.js

```diff
@@ -67,7 +67,13 @@ function PinoColada () {
     if (contentLength != null) output.push(formatBundleSize(contentLength))
     if (responseTime != null) output.push(formatLoadTime(responseTime))
 
-    return output.filter(noEmpty).join(' ')
+    var str = output.filter(noEmpty).join(' ')
+
+    // if error, output stack trace on new line
+    var err = obj.err
+    if (err && err.stack) str += nl + err.stack
```

Comment:
ahhhhh, getchu. So I think we should print the stack on both `error` and `fatal`. I was going to do that to begin with, but I changed my mind and I am not sure why... SO let's fix it. How about something like this in `formatMessage()`

```js
function formatMessage (obj) {
     if (obj.level === 'error') return chalk.dim.red(formatError(obj))
     if (obj.level === 'trace') return chalk.dim.white(obj.message)
     if (obj.level === 'warn') return chalk.dim.magenta(obj.message)
     if (obj.level === 'debug') return chalk.dim.yellow(obj.message)
     if (obj.level === 'fatal' || obj.level === 'error') return chalk.bgRed(formatError(obj) 
     if (obj.level === 'info' || obj.level === 'userlvl') return formatMessageName(obj.message)
   }

  function formatError (obj) {
    return obj.stack ? obj.message + nl + obj.stack : obj.message
  }
```